packages/gatsby/src/components/syntax.tsx

```diff
@@ -222,11 +224,11 @@ export type ArrayProps = KeyProps & {
 
 export const Array = ({theme, name, suffix, anchorTarget, children}: PropsWithChildren<ArrayProps>) => <div>
   <div>{name && <><Key theme={theme} name={name} anchorTarget={anchorTarget} /></>}{theme.arrays.leading}</div>
-  <div style={{paddingLeft: `2em`}}>
+  <div style={{position: `relative`, paddingLeft: `2em`, top: !(children as any).every((child: any) => child.type === SymlScalar) ? `2em` : ``}}>
```

Comment:
I wrote this line, it's very ugly, but I don't have the bandwidth to fix that at the moment 😢